pkg/FontChooser/app/app.js

```diff
@@ -7,6 +7,7 @@
  * https://developers.google.com/open-source/licenses/bsd
  */
 import {Runtime, Params, Chef, Decorator, Surfaces, Services, logFactory, pathForKind} from '../../arcs-import.js';
+import {loadIbis, best_solutions_to_json} from 'https://project-oak.github.io/arcsjs-provable/ibis/ibis.js';
```

Comment:
Just a note to capture what we discussed in person, we should consider a future change to follow local vs cdn mode for ibis, like we do with arcsjs.